### DISCUSSION TITLE
Setting up Sentry, getting too many issues, 95% of them are either Bad state or Null check operator - How to resolve?

### DISCUSSION THREAD

**Discussion by tahashmalik** [getsentry/sentry-dart#1836] [Q&A]
Hello Everyone,

New to Sentry, just installed it for one of our projects. We got ~7K issues in a week and I am trying to understand if these are critical errors or what. All of these issues are shown as Handled.

80% of my issues are **minified:iW: Bad state: Too many elements**

15% of them are **minified:GI: Null check operator used on a null value**

Are these serious issues? How do I go about solving them. For e.g Null Check Operator used on Null Value, we already have a Null Check in the code before the desired action, and Sentry is picking that Null Check Operator and recommending the same logic to fix it that we have applied. Not sure what we are missing here. 

Hoping for some guidance and help.

Thanks,
Taha

**Comment by buenaflor** [comment]
Hi, if you like you can send me the link of those issues at giancarlo.buenaflor@sentry.io 

It sounds like they are coming from the flutter or dart framework though

**Reply by tahashmalik** [reply]
Thanks Giancarlo, sending an email to seek further support.
